src/expr/node_traversal.cpp

```diff
@@ -63,18 +63,19 @@ TNode& NodeDfsIterator::operator*()
   return d_current;
 }
 
-bool NodeDfsIterator::operator==(const NodeDfsIterator& other) const
+bool NodeDfsIterator::operator==(const NodeDfsIterator& other)
 {
   // The stack and current node uniquely represent traversal state. We need not
   // use the scheduled node set.
   //
   // Users should not compare iterators for traversals of different nodes, or
   // traversals with different skipIfs.
+  initializeIfUninitialized();
```

Comment:
Do the comments above this line correspond to it or to the line after it?

Reply:
The line after. I'll clean this up.